### PR TITLE
Style overhaul

### PR DESCRIPTION
**Pull request by edavis25**
Significant changes to the products index view. Also switched around the search bars/pagination so they match on the index pages for products, orders, and customers. Also added some margin fixes for form-group classes and added top/bottom margin for heading elements (h1-h3).

### CODE REVIEW

File: laravel/public/css/framework.css [hunk 1]
```diff
@@ -266,7 +266,7 @@
  * Forms
  */
  div.form-group {
-
+    padding-left: 0px; /* Remove left pad so form groups sit even on left side w/ other elements */
```
Comment (edavis25) [inline laravel/public/css/framework.css:269]: So the form groups aren't set slightly to the right of other elements (ie. the index view search bars and h1 tags are now even

File: laravel/public/css/style.css [hunk 2]
```diff
@@ -158,10 +166,10 @@ section h3 {
   padding:0;
 }
 
-  p {
+p {
   margin-top: 5px;
   margin-bottom: 5px;
-  }
+}
```
Comment (edavis25) [inline laravel/public/css/style.css:172]: Indentation fix

File: laravel/resources/views/about.blade.php [hunk 3]
```diff
@@ -12,7 +12,7 @@
...
+                        Our mission is to build lasting customer relationships by delivering only the highest quality products and services that exceed the expectations of our clients. For over 50 years, we have believed transparency is key to establishing and maintaining customer satisfaction. Whether you are in the market for a new PC or a single component, no order is too large nor too small for our dedicated staff. At Lugnutz Computer Parts, your satisfaction is our number one priority, and we encourage all of our customers (potential and returning) to reach out for honest support and recommendations, or to simply learn more about Larry's legacy!
```
Comment (edavis25) [inline laravel/resources/views/about.blade.php:15]: Simple grammar fix

File: laravel/resources/views/customers/index.blade.php [hunk 4]
```diff
@@ -3,19 +3,20 @@
...
-    <div class="container">
-        {!! Form::open(['route' => 'customers.index', 'method' => 'get']) !!}
-            <div class="grid">
-              <div class="col-sm-12 form-group">
-                  {{ Form::text('search', null, array('placeholder' => 'Search customers by name...') ) }}
-                  {{ Form::submit('Search Customers') }}
-              </div>
-          </div>
-        {!! Form::close() !!}
-    </div>
```
Comment (edavis25) [inline laravel/resources/views/customers/index.blade.php:18]: Moved search form above pagination

**Comment by mcglonelevi** [comment]
Merged

**Pull request merged**